modules/sagemaker/sagemaker-templates-service-catalog/templates/multi_account_basic/product_stack.py

```diff
@@ -41,37 +66,23 @@ def __init__(
             description="Service generated Id of the project.",
         ).value_as_string
 
-        preprod_account_id = CfnParameter(
-            self,
-            "PreprodAccountId",
-            type="String",
-            description="Pre-prod account id.",
-        ).value_as_string
```

Comment:
Could we still include these parameters, only offering default values? This would allow the template user to overwrite these value.

Reply:
Yes, we can pass down the default parameter values from the props of the module - this is how it was few commits back... I thought we can hide this setup from data scientists completely but I agree there still might be a use case where this may need to be changed. 

One ugly consequence is that if the users want no VPC in dev, they would still have to provide some dummy value since there are no optional parameters in CFN.

Since we're here: another change I've been contemplating is possibly moving endpoint configs out as [seed-farmer data files](https://seed-farmer.readthedocs.io/en/latest/manifests.html#a-word-about-datafiles). What do you think?

Reply:
My view is we should leave it without the parameters. Someone using the template to create a project might not know these values and can lose interest using the template as they feel it is complex to use